doc/user_guide/introduction.rst

```diff
@@ -0,0 +1,104 @@
+Introduction
+============
+
+Welcome to the Sales Channel Module of Tryton. Simply speaking, a sales
+channel is a way of intorduting products and services to the market
+so that they get the attention of potential buyers. A sales channel can be of
+different types; 1) Direct sales channel if a business sells directly to
+customers and 2) Indirect, if there are intermediary bodies in between,
+such as retailers and dealers who are involved in selling products to
+customers.
+
```

Comment:
The sale channel module provides a way to set optimal defaults for creating orders from different channels. Examples of channels are

1. Your web store
2. Showroom 1
3. Showroom 2
4. Tradeshow 1

If you have different warehouse, pricelists, payment terms or invoicing methods that change with the channel, you could use this module to make life easier.

In addition, the sale channel module is also used by other Tryton modules that integrate with external channels like eCommerce systems (shopify, magento) and marketplaces (amazon, ebay).